keyboards/gmmk/pro/rev1/ansi/keymaps/andrebrait/config.h

```diff
@@ -16,6 +16,10 @@
 
 #pragma once
 
+#ifdef QMK_KEYS_PER_SCAN
+    #undef QMK_KEYS_PER_SCAN
+#endif
+
```

Comment:
Deprecated. 
```suggestion
```

Reply:
Was this define deprecated? I'm kinda protecting my own keymap from changes done in the lower precendence `config.h` files.

Reply:
Yes; all keys are processed in a single iteration of main loop, now.